### ISSUE TITLE
should optimize photos to display

### ISSUE THREAD

**Issue by hoatle** [HumanityOnline/MyRefuge#12]
currently, we haven't resized it to a properly size it, we need to store the original and create resized photos as we wish basing on the original photos.

This applied to the profile photos, too. Currently, we only saved resized photos (very small size), now if we need a bigger one, we can't.

So:
- store original photos
- resize as we wish basing on the saved original photos

**Comment by cazgp** [comment]
Yep, photologue is the best solution I've encountered for this. I think it's split out now into imagekit and photologue, so we probably only need imagekit (if that's what it's called).
